spec/data/record_loader/primer_panels/two_entry_example.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Did you want these example files hanging around?

Reply:
Yep, used by the tests. I'll rename the file though (future versions of record loaded already name them after the class) as its clearly confusing. (Andrew got similarly confused)